sklearn/linear_model/tests/test_sag.py

```diff
@@ -464,6 +464,7 @@ def test_sag_regressor():
     y = 0.5 * X.ravel()
 
     clf1 = Ridge(tol=tol, solver='sag', max_iter=max_iter,
+                 fit_intercept=False,
```

Comment:
I would revert changes to test if these are not necessary. Just catch the warning if need be.

Reply:
I reverted them, there is no warning because in those tests the tol and max_iter used are not the default ones